src/applications/static-pages/covid-vaccine-updates-cta/widget.jsx

```diff
@@ -10,19 +10,17 @@ import LoadingIndicator from '@department-of-veterans-affairs/component-library/
 const copy = {
   en: {
     appTitle: 'Covid 19 Vaccination Information',
-    cta: `Sign up for an easy way to stay informed about getting a COVID-19
-      vaccine at VA.`,
     expandedEligibilityContent: {
       veteran: {
         boldedNote: `If you're a Veteran currently receiving care through VA,`,
         body: ` we'll ask about your vaccine plans when you sign up. Your local VA health facility may use this information to determine when to contact you once your risk group becomes eligible.`,
       },
       nonVeteran: {
-        boldedNote: `If you're a Veteran, spouse, or caregiver not receiving care through VA,`,
-        body: ` sign up to tell us if you want to get a vaccine through VA. If you're eligible, we'll contact you when we have a vaccine available for you. At this time, we don't know when that will be.`,
+        boldedNote: `If you're a Veteran who isn’t receiving care through VA or a spouse, caregiver, or CHAMPVA recipient`,
```

Comment:
Nit - but `you're` is using a `'` while `isn’t` uses the proper `’`. Totally optional but possibly update the former to be `you’re`.

Reply:
Fixed! Eagle eyes Sullivan!